Kernel/VM/Region.cpp

```diff
@@ -228,6 +231,7 @@ void Region::map_individual_page_impl(size_t page_index)
         pte.set_physical_page_base(0);
         pte.set_present(false);
     } else {
+        pte.set_cache_disabled(!m_cacheable);
```

Comment:
This change means that basically all memory now becomes uncacheable.
It's good that you're adding the ability to control this flag, but the `cacheable` argument to the `Region` construction helpers should really be `true` by default. If you declare those functions with `cacheable = true`, you also don't have to update the callers, making the patch a bit smaller :)

Reply:
that's actually a good idea :)